### ISSUE TITLE
apache down after letsencrypt-auto, how to revert back?

### ISSUE THREAD

**Issue by dobster42** [certbot/certbot#5187]
## My operating system is (include version):
Debian 7.11

## I installed Certbot with (certbot-auto, OS package manager, pip, etc):
certbot-auto

## I ran this command and it produced this output:
./letsencrypt-auto --apache -d MYDOMAIN.eu -d www.MYDOMAIN.eu 
Creating virtual environment...
Installing Python packages...
Installation succeeded.
Saving debug log to /var/log/letsencrypt/letsencrypt.log
Plugins selected: Authenticator apache, Installer apache
Obtaining a new certificate
Performing the following challenges:
tls-sni-01 challenge for MYDOMAIN.eu
tls-sni-01 challenge for www.MYDOMAIN.eu
Waiting for verification...
Cleaning up challenges
Error while running apache2ctl graceful.
httpd not running, trying to start
Action 'graceful' failed.
The Apache error log may have more information.

(98)Address already in use: make_sock: could not bind to address [::]:443
(98)Address already in use: make_sock: could not bind to address 0.0.0.0:443
no listening sockets available, shutting down
Unable to open logs

Error while running apache2ctl graceful.
httpd not running, trying to start
Action 'graceful' failed.
The Apache error log may have more information.

(98)Address already in use: make_sock: could not bind to address [::]:443
(98)Address already in use: make_sock: could not bind to address 0.0.0.0:443
no listening sockets available, shutting down
Unable to open logs

IMPORTANT NOTES:
 - The following errors were reported by the server:

   Domain: MYDOMAIN.eu
   Type:   unauthorized
   Detail: Incorrect validation certificate for tls-sni-01 challenge.
   Requested
   0e1ddf3dc0cf7efd22bbdeedb38ade01.6542df6a832e872971061719e9a6051b.acme.invalid
   from XX.214.221.4:443. Received 1 certificate(s), first certificate
   had names "parallels panel"

   Domain: www.MYDOMAIN
   Type:   unauthorized
   Detail: Incorrect validation certificate for tls-sni-01 challenge.
   Requested
   8d7dadf403ea990bdca0fdaf39432112.ae2efcaef19e9d3b1180b2172ade30af.acme.invalid
   from XX.214.221.4:443. Received 1 certificate(s), first certificate
   had names "parallels panel"

   To fix these errors, please make sure that your domain name was
   entered correctly and the DNS A/AAAA record(s) for that domain
   contain(s) the right IP address.
root@h2336913:/opt/letsencrypt# [17398:28257:1013/115656.266986:ERROR:cert_verify_proc_nss.cc(902)] CERT_PKIXVerifyCert for XXXXXXXX.stratoserver.net failed err=-8

## Certbot's behavior differed from what I expected because:
gave errors and apache does not start any more

**Comment by bmw** [comment]
What's the output of `sudo service apache2 start`? If it's still unable to obtain port 443, do you know if your Apache configuration was previously listening on this port? Do you know what's listening on this port now (possibly by using instructions provided [here](https://unix.stackexchange.com/questions/106561/finding-the-pid-of-the-process-using-a-specific-port))?

If Apache wasn't previously listening on port 443 on your server or you're not sure if it was, please provide the full log corresponding to the output above. Logs are stored in `/var/log/letsencrypt` by default. Feel free to redact domains, e-mail and IP addresses as you see fit.

**Comment by dobster42** [comment]
Thanks for your answer, Brat. nginx was listening on this port and I got
apache running again after settings the ports.? file, alas without https
support. At the moment I am completely nackered from downing a production
server for 5 hours on friday the 13th. and winding down with a bottle of
vodka. Getting https going is of utmost priority and I will follow your
provided information as soon as I am sober again :-)
Since it appears that your question makes sense from what I have found out
I believe that I will get along by reading the info provided by your link
(without having looked it now).

Best regard,

Dirk

Am 13.10.2017 21:13 schrieb "Brad Warren" <notifications@github.com>:
